### ISSUE TITLE
[BUG] [VSCode Extension]

### ISSUE THREAD

**Issue by mishrapratyush** [microsoft/promptflow#2768]
**Describe the bug**
PromptFlow vectorDB tool fails to configure in VSCode

**How To Reproduce the bug**
Steps to reproduce the behavior, how frequent can you experience the bug:
1. Create a new Flow in VSCode
2. pip install promptflow-vectordb
3. Create Connection to CognitiveSearch using admin keys
4. Add promptflow-vectordb tool to your flow canvas
5. click on the mlindex_content field to configure 

**Screenshots**
1. On the VSCode primary side bar > the Prompt flow pane > quick access section. Find the "install dependencies" action. Please it and attach the screenshots there.
![image](https://github.com/microsoft/promptflow/assets/8485494/ddbb5e20-1a9e-4124-897c-8c10b6127535)
6. Please provide other snapshots about the key steps to repro the issue.
![image](https://github.com/microsoft/promptflow/assets/8485494/2fb02083-1cfb-4560-accd-86cc036eb13e)

**Environment Information**
 - Promptflow Package Version using `pf -v`: [1.7.0]
 - Operating System: [Windows 11]
 - Python Version using `python --version`: [e.g. python==3.10.11]
 - VS Code version.
 - Prompt Flow extension version.
 - On the VS Code bottom pane > Output pivot > "prompt flow" channel, find the error message could be relevant to the issue and past them here. That would be helpful for our trouble shooting.
 - If your code to repro the issue is public and you want to share us, please share the link.

**Additional context**

**Comment by Adarsh-Ramanathan** [comment]
@mishrapratyush, you'll want to be `az login`ed on your shell, and you'll need to configure defaults for your sub/rg/workspace before attempting to configure Index Lookup:

```
az login
az account set --subscription <subscription_id>
az configure --defaults group=<resource_group_name> workspace=<workspace_name>
```

Can you try again once your environment is configured correctly, and let me know if that resolves your issue?

**Comment by mishrapratyush** [comment]
Thanks for the instructions, that worked with these tweaks:
1. Once these defaults were set, Generate mlindex_content was able to popup without errors - however it couldnt find the AzureOpenAI and CogSearch connections I had in local - it was trying to match those in the workspace. So I had to go to my AML Workspace and create connections with the same name as the ones I had in VSCode.
2. The Generate mlindex_content form - wants "acs_metadata_field" to be filled in - which i dont know what needs to go there. However, in my AML workspace - I was able to leave it blank and it didn't complain. So at the end, I created the connection in the AML Workspace and then copied the code from there to VSCode and it worked. Now I don't know if it really did hybrid search or it just did keyword search

If you want, I can create a PR to update this doc to include the above instructions: https://microsoft.github.io/promptflow/reference/tools-reference/vector_db_lookup_tool.html#requirements
